src/usecases/sendVisitsMilestoneNotification.js

```diff
@@ -0,0 +1,18 @@
+const sendVisitsMilestoneNotification = ({ getSlack }) => async ({
+  numberOfVisits,
+}) => {
+  if (numberOfVisits % 500 === 0) {
```

Comment:
Maybe the 500 should be an env variable or at least a const at the top of file. I hope that the app gets a lot of use then 500 will be too often!

Reply:
Ah good shout, I like this idea!